.changeset/strange-steaks-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': minor
+---
+
+Disable resizing for Elastic Textareas
```

Comment:
Can we also add a changeset for fixing the bug where `c-input` did not respect `rows`?

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/2098/commits/e38c51c015fbfee76cb4b49888ae820051ae2430